src/Dependencies/Collections/Microsoft.CodeAnalysis.Collections.Package.csproj

```diff
@@ -2,7 +2,7 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net6.0;netstandard2.0</TargetFrameworks>
```

Comment:
```suggestion
    <TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.0</TargetFrameworks>
```

Reply:
Build warning is suppressed for this project, we can add back the tfm

Reply:
Ok. I don't know why you'd want to target an unsupported runtime version, but I'll put it back.